gc/base/PhysicalSubArenaVirtualMemoryFlat.cpp

```diff
@@ -211,11 +219,12 @@ MM_PhysicalSubArenaVirtualMemoryFlat::expandNoCheck(MM_EnvironmentBase *env, uin
 		getHeapRegionManager()->resizeAuxillaryRegion(env, _region, _lowAddress, _highAddress);
 		Assert_MM_true(NULL != _region);
 
-		if(result){
+		if(result) {
 			genericSubSpace->addExistingMemory(env, this, expandSize, lowExpandAddress, highExpandAddress, true);
+			_subSpace->heapReconfigured(env, RECONFIG_EXPAND, genericSubSpace, lowExpandAddress, highExpandAddress);
```

Comment:
I think it would be safe to call against genericSubSpace. It would cause one more indirection in runtime (genericSubSpace calling its parent _subSpace), but this sequence would be unified.

Consider other similar spots, too.

Reply:
As discussed offline, I'll look into this more/verify and make changes in a separate PR if required